src/Microsoft.TemplateEngine.Edge/Template/WellKnownSearchFilters.cs

```diff
@@ -29,32 +30,15 @@ public static class WellKnownSearchFilters
 
                 bool hasShortNamePartialMatch = false;
 
-                if (template is FilterableTemplateInfo filterableTemplate)
-                {
-                    foreach (string shortName in filterableTemplate.GroupShortNameList)
-                    {
-                        int shortNameIndex = shortName.IndexOf(name, StringComparison.OrdinalIgnoreCase);
+                int shortNameIndex = template.ShortName.IndexOf(name, StringComparison.OrdinalIgnoreCase);
```

Comment:
Shouldn't we check `template.ShortNameList` here as well?

Reply:
that is added in https://github.com/dotnet/templating/pull/3012